From: Joe Perches <joe@perches.com>
Date: Wed, 11 Jan 2017 14:52:20 -0800
Subject: tilepro: Fix non-void return from void function
Patch-mainline: v4.11-rc1
Git-commit: b65b09aa79086f0f2aa8935eae410f87495b8c04
References: bsc#1015336 FATE#321685 bsc#1015337 FATE#321686 bsc#1015340 FATE#321687

commit bc1f44709cf2 ("net: make ndo_get_stats64 a void function")
mistakenly used a return value for this void conversion.

Fix it.

Signed-off-by: Joe Perches <joe@perches.com>
cc: stephen hemminger <stephen@networkplumber.org>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Benjamin Poirier <bpoirier@suse.com>
---
 drivers/net/ethernet/tile/tilepro.c |    3 ---
 1 file changed, 3 deletions(-)

--- a/drivers/net/ethernet/tile/tilepro.c
+++ b/drivers/net/ethernet/tile/tilepro.c
@@ -2091,11 +2091,8 @@ static void tile_net_get_stats64(struct
 	stats->tx_bytes   = tx_bytes;
 	stats->rx_errors  = rx_errors;
 	stats->rx_dropped = rx_dropped;
-
-	return stats;
 }
 
-
 /*
  * Change the "mtu".
  *
